sdk/data-generators/number-stream-generator/Cargo.toml

```diff
@@ -0,0 +1,15 @@
+[package]
+name = "number-stream-generator"
+version = "0.1.0"
+authors = ["Shale Xiong <shale.xiong@arm.com>"]
```

Comment:
"The Veracruz Development Team".

Reply:
Also: can you add a `description` field to the `Cargo.toml` file to explain what the purpose of this is?